### PR TITLE
add cse emoticons

### PR DESCRIPTION
**Pull request by JoelTerMeer**
Added the CSE custom emoticons, and changed regex to match mehuge/sysrage recommendation to allow :pvp: and :plus10: to work.

### CODE REVIEW

File: patcher/src/ts/content/chat/ParseEmoji.tsx [hunk 1]
```diff
@@ -8,15 +8,15 @@ import * as React from 'react';
 
 function emojiNameFromText(text:string): string {
   switch(text) {
-    case ':angry:': case ':[': case ':-[':
```
Comment (sysrage) [inline patcher/src/ts/content/chat/ParseEmoji.tsx]: Why'd you remove ":[" and ":-[" matches from this?

File: patcher/src/ts/content/chat/ParseEmoji.tsx [hunk 2]
```diff
@@ -8,15 +8,15 @@ import * as React from 'react';
...
+    case ':angry:': 
       return 'angry';
-    case ':baffled:': case 'O.o':
+    case ':baffled:': case 'O.o':
```
Comment (sysrage) [inline patcher/src/ts/content/chat/ParseEmoji.tsx]: Remove the extra space.

**Comment by sysrage** [comment]
Tested this and all listed emotes seem to work well. Besides prior comments, only other suggestion is to add a few additional names for the art, tdd, and viking specific emotes:

:artlove:
:tddlove:
:viklove:
:artshield:
:tddshield:
:vikshield:

**Comment by JoelTerMeer** [comment]
Closing to make recommended changes.

**Pull request closed without merging**